README.md

```diff
@@ -297,7 +297,27 @@ int7 = 1_2_3_4_5     # VALID but discouraged
 ```
 
 Leading zeros are not allowed. Integer values `-0` and `+0` are valid and
-identical to an unprefixed zero. Hex, octal, and binary forms are not allowed.
+identical to an unprefixed zero.
+
+Integer values may also be expressed in hexadecimal, octal, or binary. Hex
+values are case insensitive. Leading zeros are allowed. Underscores are allowed
```

Comment:
Might want to clarify that the "Leading zeros are allowed" sentence here applies only to prefixed hex, octal, and/or binary literals. Otherwise there appears to be a contradiction in the spec between "Leading zeros are not allowed" in the previous paragraph and "Leading zeros are allowed" in this paragraph.

Reply:
Clarified in the commit I just pushed.